Improve Button TW component (#6946)
plone/volto: docs/upgrade-guide/seven.md

---
myst:
  html_meta:

>     "description": "This upgrade guide lists all breaking changes in Seven and explains the steps that are necessary to upgrade to the latest version."
>     "property=og:description": "This upgrade guide lists all breaking changes in Seven and explains the steps that are necessary to upgrade to the latest version."
>     "property=og:title": "Upgrade Guide"
>     "keywords": "Seven, Plone, frontend, React, Upgrade, Guide"

Reviewer: ```suggestion
    "description": "This upgrade guide lists all breaking changes in Seven, and explains the necessary steps to upgrade your add-on for the latest version."
    "property=og:description": "This upgrade guide lists all breaking changes in Seven, and explains the necessary steps to upgrade your add-on for the latest version."
    "property=og:title": "Seven upgrade guide"
    "keywords": "Seven, Plone, frontend, React, upgrade, guide"
```

---

(seven-upgrade-guide)=

> # Upgrade Guide

Reviewer: ```suggestion
# Upgrade guide
```

> This upgrade guide lists all breaking changes in Seven and explains the steps that are necessary to upgrade to the latest version.

Reviewer: ```suggestion
This upgrade guide lists all breaking changes in Seven, and explains the necessary steps to upgrade your add-on for the latest version.
```

> Seven uses Semantic Versioning.
> For more information see {doc}`../contributing/version-policy`.

Reviewer: ```suggestion
Seven uses Semantic Versioning, as described in {doc}`../contributing/version-policy`.
```
